autogen/docs/interfacetab.md.mustache

```diff
@@ -306,6 +306,8 @@ To return the observer and focus point to their default positions press the
 
 #### Fullscreen Mode
 
+Fullscreen mode makes the 3D view of NetLogo Desktop fit on the entire screen. It is often useful for exhibits such as museum exhibits.
```

Comment:
Seems weird to use 'exhibits' twice there.  Maybe "It is often useful for kiosk displays such as museum exhibits"?